doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -153,6 +153,11 @@ HomeKit
 
 HomeKit is now removed, as announced in the :ref:`ncs_release_notes_250`.
 
+Core firmware
+=============
+
+* Added the :ref:`ipc_radio` firmware image.
```

Comment:
This could rather be mentioned in the Documentation section like this:
* Added :ref:`core_fw` documentation.